### ISSUE TITLE
Playlist files are not visible in Library.

### ISSUE THREAD

**Issue by parkmino** [yktoo/ymuse#2]
Library does not show playlist files, e.g., m3u and pls.

**Comment by yktoo** [comment]
That's right, they're filtered out. Do you think it's useful to show them there?

**Comment by parkmino** [comment]
I do not have playlist_directory in mpd.conf, it would be helpful for me.

**Comment by yktoo** [comment]
Showing playlists has been added in `dev`, but full support for loading playlists is unfortunately not possible yet. MPD requires specifying the absolute file path which I cannot obtain with the current version of `gompd` (it requires the `config` command).

Also, MPD only supports `.m3u` playlists out-of-the-box, `.pls` support requires a plugin.

**Comment by parkmino** [comment]
Thanks and I will try this release !

**Comment by parkmino** [comment]
Now m3u and pls playlists are visible in library, however it show the following error message.
They are all http streams.

![2020-05-21-173333_452x132_scrot](https://user-images.githubusercontent.com/12131158/82540093-7db41480-9b89-11ea-99e8-3a67f1c60456.png)

**Comment by yktoo** [comment]
Exactly, that the problem I described. MPD expects an absolute path which I cannot determine with the current version of `gompd`.

Let's keep this open until `config` is implemented there.
